src/OpenTelemetry.Exporter.Prometheus/PrometheusExporter.cs

```diff
@@ -39,10 +39,15 @@ public PrometheusExporter(PrometheusExporterOptions options)
 
         internal Action CollectMetric { get; set; }
 
-        public override ExportResult Export(IEnumerable<Metric> metrics)
+        public override ExportResult Export(in Batch<Metric> metrics)
         {
             this.Metrics = metrics;
             return ExportResult.Success;
         }
+
+        public override AggregationTemporality GetAggregationTemporality()
```

Comment:
curious - does this have to be `GetAggregationTemporality()` or it can be a property with public get and protected set?

Reply:
yes i think it can be property. will adjust this. (not part of the coming release., but after that)